benchmarks/appsec_iast_django_startup/requirements_scenario.txt

```diff
@@ -0,0 +1,3 @@
+django==4.2.5
```

Comment:
you might want to generate a lock file for all the deps here. that way if a dep of a dep changes the performance impact. lock it down.

mild suggestion, can be done as a follow-up